libraries/AP_HAL_ChibiOS/SPIDevice.cpp

```diff
@@ -426,11 +426,11 @@ void SPIDevice::test_clock_freq(void)
     hal.console->printf("Waiting for USB\n");
     for (uint8_t i=0; i<3; i++) {
         hal.scheduler->delay(1000);
-        hal.console->printf("Waiting %u\n", AP_HAL::millis());
+        hal.console->printf("Waiting %lu\n", AP_HAL::millis());
```

Comment:
```suggestion
        hal.console->printf("Waiting %u\n", (unsigned)AP_HAL::millis());
```

Reply:
@peterbarker Thanks! I have just updated it